Cargo.lock

```diff
@@ -0,0 +1,717 @@
+# This file is automatically @generated by Cargo.
```

Comment:
`Cargo.lock` should not be added to library crates.

Reply:
Oops, embarrassing!